dist/components/tabs/_tabs.dust

```diff
@@ -0,0 +1,20 @@
+{?tabs}
+    <div class="c-tabs" role="tab">
+        <ul id="c-tab-controls" class="c-tabs__controls" role="tablist">
+            {#tabs}
+                <li class="c-tabs__controls-item {class}">
+                    <button class="c-tabs__button" data-target="{id}" aria-label="{title}">{title}</button>
```

Comment:
Hmm, is that still a requirement? I can't remember why that's needed. Doesn't it just default to `type="button"` if it isn't declared?